src/main/resources/js/condec.context.menu.js

```diff
@@ -12,251 +12,286 @@
+            document.getElementById("condec-context-menu-link-item").style.display = "initial";
+            document.getElementById("condec-context-menu-delete-link-item").style.display = "initial";
+            document.getElementById("second-context-section").style.display = "block";
+            document.getElementById("third-context-section").style.display = "block";
+            document.getElementById("fourth-context-section").style.display = "block";
+        }
+        if (documentationLocation !== "groups") {
+            conDecAPI.getDecisionKnowledgeElement(id, documentationLocation, function (decisionKnowledgeElement) {
+                var typesForStatus = new Array("issue", "alternative", "decision");
+                if (!typesForStatus.includes(decisionKnowledgeElement.type.toLowerCase())) {
+                    document.getElementById("condec-context-menu-change-status-item").style.display = "none";
+                } else {
+                    document.getElementById("condec-context-menu-change-status-item").style.display = "initial";
+                }
+            });
+        }
+        if (container == null || container.includes("vis")) {
+            document.getElementById("condec-context-menu-set-root-item").style.display = "none";
+            document.getElementById("condec-context-menu-delete-link-item").style.display = "none";
+        } else if (documentationLocation !== "c") {
+            document.getElementById("condec-context-menu-set-root-item").style.display = "initial";
+            document.getElementById("condec-context-menu-delete-link-item").style.display = "initial";
+        }
+
+        if (documentationLocation === "s") {
+            document.getElementById("condec-context-menu-sentence-irrelevant-item").style.display = "initial";
+            conDecAPI.isIssueStrategy(function (isEnabled) {
+                if (isEnabled) {
+                    document.getElementById("condec-context-menu-sentence-convert-item").style.display = "initial";
+                } else {
+                    document.getElementById("condec-context-menu-sentence-convert-item").style.display = "none";
+                }
+            });
+            document.getElementById("condec-context-menu-set-root-item").style.display = "none";
+        } else {
+            document.getElementById("condec-context-menu-sentence-irrelevant-item").style.display = "none";
+            document.getElementById("condec-context-menu-sentence-convert-item").style.display = "none";
+            document.getElementById("condec-context-menu-set-root-item").style.display = "initial";
+        }
+        /*if (documentationLocation === "i"){
+            document.getElementById("condec-context-menu-assign-decision-group-item").style.display = "none";
+        }else{
+            document.getElementById("condec-context-menu-assign-decision-group-item").style.display = "initial";
+        }*/
+        if (documentationLocation == "groups") {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5229259)